packages/v4/src/content/design-guidelines/components/context-selector/context-selector.md

```diff
@@ -2,15 +2,59 @@
 id: Context selector
 section: components
 ---
-A **context selector** can be used in addition to global navigation when the data or resources you show in the interface need to change depending on the users’ context.
+A **context selector** can be used in addition to global navigation when the data or resources you show in the interface need to change depending on the user’s context.
 
 ## Usage
 
-Context selectors enable a user to switch between different application contexts, like administrator and regular user view, for example.
+Context selectors enable a user to switch between different application contexts. They can be placed at many levels of an application’s information architecture and may have different behavior expectations based on the level of the IA it is placed in. They can be placed in a page masthead, at the top of vertical navigations, or at the top of a page, depending on your website or product's navigation and preference.
+ 
+Context selectors differ from filters in that a filter takes an overall set of content and reduces it to a subset of content.  In comparison, a context selector takes a set of content and changes it to a different set of content.
 
-They can be placed in the page masthead, or at the top of vertical navigations, depending on your website or product's navigation and preference.
+## Application-wide impact
+When a context selector has an application-wide impact the user is expected to choose a context that will set the content of the entire application.  The list of choices inside the selector's menu can be dynamic and long, but the navigation for any of those choices should not change.
+ 
+### Use cases 
+Some common use cases for using a context selector with an application-wide impact include:
+* Switching between applications or tools that are part of a product suite where the list of choices is known, and is not dynamic.
+* Setting the content scope of the entire application, and all navigation areas within the application now following the content scope. 
 
-<img src="./img/context-selector.png" alt="Example of context selector inside the masthead " width="990"/>
+### Recommended placement
+It is recommended that a context selector that has an application-wide impact be placed in the masthead, above the main navigation. This creates an effect that the entire application is changing, rather than just changing the views or navigation within an application.  
 
+<img src="./img/application-wide.png" alt="Example of a context selector with application wide impact, placed in the masthead" width="990"/>
```

Comment:
I wonder if we should make this match the latest masthead mocks. This is currently under development, but reflects where we want to go. https://marvelapp.com/prototype/4d9chi8/screen/67194844

Reply:
@mcarrano Do you know when this will be released? I'm thinking I can always edit it later - along with every other image that uses a masthead in the design guidelines